src/components/selector/selector.ts

```diff
@@ -0,0 +1,44 @@
+import './selector.scss';
+import WebComponent, { createElementFromString } from '../../common/WebComponent';
+
+class SelectorComponent implements WebComponent {
+  dropdownMenu: HTMLSelectElement;
+  options: HTMLOptionsCollection;
+
+  constructor(dropdown: HTMLSelectElement) {
```

Comment:
We want reusable and universal component. To do so I need to have possibility to specify a list with values to select and with value if nothing is selected yet. I also need possibility to change width of this component.